docs/best-practices/serverless/lambda.md

```diff
@@ -53,7 +53,7 @@ The  [README in the example repo has more details regarding the Lambda Setup.](h
 [(You can get your Server KE the server-side SDK key from the DevCycle dashboard)](/essentials/keys)
 
 ```jsx
-const dvcClient = await DVC.initialize('<DEVCYCLE_SERVER_SDK_KEY>').onClientInitialized()
+const devcycleClient = await DVC.initialize('<DEVCYCLE_SERVER_SDK_KEY>').onClientInitialized()
```

Comment:
Same thing here, `DVC` is coming from the https://github.com/DevCycleHQ/devcycle-lambda-example/ - task for a future day